src/resolvers/react.ts

```diff
@@ -56,41 +57,29 @@ export async function resolveReactRoutes(ctx: PageContext) {
 
   pageRoutes.forEach((page) => {
     const pathNodes = page.route.split('/')
-
     const element = page.path.replace(ctx.root, '')
-
     let parentRoutes = routes
 
     for (let i = 0; i < pathNodes.length; i++) {
       const node = pathNodes[i]
-      const isDynamic = isDynamicRoute(node, nuxtStyle)
-      const isCatchAll = isCatchAllRoute(node, nuxtStyle)
-      const normalizedName = isDynamic
-        ? nuxtStyle
-          ? isCatchAll ? 'all' : node.replace(/^_/, '')
-          : node.replace(/^\[(\.{3})?/, '').replace(/\]$/, '')
-        : node
-      const normalizedPath = normalizeCase(normalizedName, caseSensitive)
 
       const route: Route = {
+        caseSensitive,
```

Comment:
We should cleanup this in prepare route function, otherwise this will include in generated routes.

Reply:
`caseSensitive` is supported by react router v6 by default https://reactrouterdotcom.fly.dev/docs/en/v6/api#routes-and-route

```ts
interface RouteProps {
  caseSensitive?: boolean;
  children?: React.ReactNode;
  element?: React.ReactElement | null;
  index?: boolean;
  path?: string;
}
```

Reply:
Oh... I didn't know that, haha